src/components/access_group/AccessGroupCurators.vue

```diff
@@ -0,0 +1,141 @@
+<template>
```

Comment:
Let's confirm if we still need this (it's not used right now and I'm not sure which designs are up to date).

Reply:
We don't exactly. I just have been torn about whether to actually delete it.